README

```diff
@@ -2859,12 +2859,12 @@ compressed sound. For compressed save states, zlib is required.
 
 Some parts of ScummVM, particularly scalers, have highly optimized
 versions written in assembler. If you wish to use this option, you will
-need to install nasm assembler (see <http://www.nasm.us/>). Note that
+need to install nasm assembler (see <https://www.nasm.us/>). Note that
 we currently only have x86 MMX optimized versions, and they will not
 compile on other processors.
 
 On Win9x/NT/XP, you can define USE_WINDBG and attach WinDbg to browse
-debug messages (see <https://technet.microsoft.com/en-us/sysinternals/debugview.aspx>).
+debug messages (see <https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/index>).
```

Comment:
Hm, here we refer to `WinDbg` but the current (old) URL links to `DebugView` instead - why is that? Do we want to keep the link to `DebugView` (in which case the URL should be updated to https://docs.microsoft.com/en-us/sysinternals/downloads/debugview instead) or should we go with the proposed change which actually does link to `WinDbg`? In the former case the whole sentence needs updating.

Reply:
It was added in 2002: https://github.com/scummvm/scummvm/commit/965b96b4895f0a5ad203abd0d9b48cc438447b31

And it looks that it should, in fact, originally say "WinDbg or similar", and then link to DebugView which is standalone, lightweight and not requiring SDK download.

On the other hand, I am not sure it is a good idea to link to a WinXP-era utility for the Windows 10 world. So, I'd change it to WinDbg link as this patch suggests.

Reply:
Right. I've now tried and can't get DebugView to work on Windows 10 (apparently it's supposed to install `dbgv.sys` but it never ends up in my driver directory and all options relation to capture are greyed-out) which is unfortunate because it's a small, standalone tool while `WinDbg` requires a download and installation of at least ~500 MB worth of stuff.